specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -2656,36 +2885,49 @@
           },
           "description": "A collection of reply URLs for the application."
         },
-        "keyCredentials": {
+        "requiredResourceAccess": {
           "type": "array",
           "items": {
-            "$ref": "#/definitions/KeyCredential"
+            "$ref": "#/definitions/RequiredResourceAccess"
           },
-          "description": "The list of KeyCredential objects."
+          "description": "Specifies resources that this application requires access to and the set of OAuth permission scopes and application roles that it needs under each of those resources. This pre-configuration of required resource access drives the consent experience."
         },
-        "passwordCredentials": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/PasswordCredential"
-          },
-          "description": "The list of PasswordCredential objects."
+        "samlMetadataUrl": {
+          "description": "The URL to the SAML metadata for the application.",
+          "type": "string"
         },
-        "oauth2AllowImplicitFlow" : {
-          "type": "boolean",
-          "description": "Whether to allow implicit grant flow for OAuth2"
+        "signInAudience": {
+          "description": "Audience for signing in to the application (AzureADMyOrganizatio, AzureADAllorganizations, AzureADAndMicrosofAccounts).",
+          "type": "string"
         },
-        "requiredResourceAccess" : {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/RequiredResourceAccess"
-          },
-          "description": "Specifies resources that this application requires access to and the set of OAuth permission scopes and application roles that it needs under each of those resources. This pre-configuration of required resource access drives the consent experience."
+        "wwwHomepage": {
+          "description": "The primary Web page.",
+          "type": "string"
         }
-      },
-      "additionalProperties": {
-        "type": "object"
-      },
-      "description": "Request parameters for updating an existing application."
+      }
+    },
+    "ApplicationCreateParameters": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ApplicationBase"
+        }
+      ],
+      "required": [
+        "availableToOtherTenants",
```

Comment:
It's not supporting by Swagger (required from a allOf):
To do that, you will need to duplicate the three in both Update and Create models, and make them required in Create,